scripts/install

```diff
@@ -64,42 +73,33 @@ db_pwd=$(ynh_app_setting_get --app=$app --key=mysqlpwd)
 ynh_script_progression --message="Setting up source files..." --weight=4
 
 ynh_app_setting_set --app=$app --key=final_path --value=$final_path
-
-# In case of a new version, the url change from http://download.dotclear.org/latest/dotclear-X.X.X.tar.gz to http://download.dotclear.org/attic/dotclear-X.X.X.tar.gz
-src_url=$(grep 'SOURCE_URL=' "../conf/app.src" | cut -d= -f2-)
-if ! curl --output /dev/null --silent --head --fail "$src_url"; then
-	ynh_replace_string --match_string="latest" --replace_string="attic" --target_file="../conf/app.src"
-fi
-
+# Download, check integrity, uncompress and patch the source from app.src
```

Comment:
```suggestion

# In case of a new version, the url change from http://download.dotclear.org/latest/dotclear-X.X.X.tar.gz to http://download.dotclear.org/attic/dotclear-X.X.X.tar.gz
src_url=$(grep 'SOURCE_URL=' "../conf/app.src" | cut -d= -f2-)
if ! curl --output /dev/null --silent --head --fail "$src_url"; then
	ynh_replace_string --match_string="latest" --replace_string="attic" --target_file="../conf/app.src"
fi

# Download, check integrity, uncompress and patch the source from app.src
```